idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -22,18 +22,21 @@
 LOGS_OUT_BASENAME = 'log'
 STATS_OUT = 'stats.json'
 VERSION_OUT = 'versions.json'
+PIPELINE_VERSION_OUT = 'pipeline_version.txt'
 
 # arguments from environment variables
 INPUT_BUCKET = os.environ.get('INPUT_BUCKET')
 FILE_TYPE = os.environ.get('FILE_TYPE')
-OUTPUT_BUCKET = os.environ.get('OUTPUT_BUCKET')
+OUTPUT_BUCKET = os.environ.get('OUTPUT_BUCKET').rstrip('/')
 STAR_GENOME = os.environ.get('STAR_GENOME', 's3://czbiohub-infectious-disease/references/human/STAR_genome.tar')
 BOWTIE2_GENOME = os.environ.get('BOWTIE2_GENOME', 's3://czbiohub-infectious-disease/references/human/bowtie2_genome.tar')
 STAR_BOWTIE_VERSION_FILE_S3 = os.environ.get('STAR_BOWTIE_VERSION_FILE_S3', get_host_index_version_file(STAR_GENOME))
 DB_SAMPLE_ID = os.environ['DB_SAMPLE_ID']
 AWS_BATCH_JOB_ID = os.environ.get('AWS_BATCH_JOB_ID', 'local')
+PIPELINE_VERSION = os.environ.get('PIPELINE_VERSION')
+SAMPLE_S3_OUTPUT_POSTFIX = "/%s" % major_version(PIPELINE_VERSION) if PIPELINE_VERSION else ""
```

Comment:
I think making the version optional is probably more trouble than it's worth.

Reply:
need it for idseq-web compatibility. But I can remove it afterwards.